tests/test_geos_err_handler.py

```diff
@@ -2,11 +2,16 @@
 
 import pytest
 
+import pygeos
+
 from shapely.geometry import LineString
 from shapely.errors import ReadingError
 from shapely.wkt import loads
 
+from tests.conftest import shapely20_todo
+
 
+@shapely20_todo  # logging is not yet implemented
```

Comment:
This is something we need to check in pygeos how this can be possible, if it's a feature we want to preserve

Reply:
I'd say all error messages from GEOS (like the one below) should be converted to exception messages, similar to https://github.com/Toblerity/Shapely/issues/991#issuecomment-699147566 . I'd assume it would be difficult to capture an error message in pygeos, but also emit the same error message to the logger handler below. I wouldn't expect many other folks to be relying on these log capture methods.

Reply:
Opened https://github.com/Toblerity/Shapely/issues/998 for a discussion on logging.